mmtk/Cargo.toml

```diff
@@ -29,7 +29,7 @@ once_cell = "1.10.0"
 # - change branch
 # - change repo name
 # But other changes including adding/removing whitespaces in commented lines may break the CI.
-mmtk = { git = "https://github.com/mmtk/mmtk-core.git", rev = "4904004d55c48858d12a97a8e2ebb559fb91c0de" }
+mmtk = { git = "https://github.com/qinsoon/mmtk-core.git", rev = "dc2a4293d103ca814f5f9e526552b5ef08502240" }
```

Comment:
Will this be reverted after the PR is merged?

Reply:
Yeah. This will be updated once we merge the mmtk-core PR.